client-mvp-04/src/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+@import '../variables';
+
+.dropdown-container {
```

Comment:
Class `.dropdown-container` is related just to `DevUiKit`. 
Inside the `Dropdown `component, we don't have this wrapper.
So, you need to move these styles to `uiKit.scss.`